lib/Chat/Parser/SystemMessage.php

```diff
@@ -119,6 +119,20 @@ public function parseMessage(Message $chatMessage): void {
 			} elseif ($cliIsActor) {
 				$parsedMessage = $this->l->t('An administrator renamed the conversation from "%1$s" to "%2$s"', [$parameters['oldName'], $parameters['newName']]);
 			}
+		} elseif ($message === 'description_set') {
+			$parsedMessage = $this->l->t('{actor} set the description to "%1$s"', [$parameters['newDescription']]);
```

Comment:
I would not put the description into the system message.
It can be potentially long and also contain `"` and then look broken.

Whatsapp and others just write "{actor} updated the description" alike messages.

Reply:
And we can then use this for create, update and remove at the same time.

Reply:
@ma12-co checked other platforms and asked for the description to be shown in the message.

Reply:
I think it should be shown, given that:
1) It's likely to be very important information for the conversation participants;
2) It's s going to be relatively concise since we're capping the max length to 500 chars, so it's not going to be disruptive;